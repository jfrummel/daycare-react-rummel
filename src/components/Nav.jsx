import React from 'react';
import { NavLink } from 'react-router-dom';


const Nav = () => (
    <nav>
        <NavLink exact to="/" activeClassName="activeLink">Home</NavLink>
        <NavLink to="/about" activeClassName="activeLink">About</NavLink>
        <NavLink to="/contact" activeClassName="activeLink">Contact</NavLink>
    </nav>
);

export default Nav;