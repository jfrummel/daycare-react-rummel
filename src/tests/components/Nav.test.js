import React from 'react';
import { shallow } from 'enzyme';
import Nav from '../../components/Nav'

test("renders nav componet", () => {
    const wrapper = shallow(<Nav />);
    expect(wrapper).toMatchSnapshot();
});