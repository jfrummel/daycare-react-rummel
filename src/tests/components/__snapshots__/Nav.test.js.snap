// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders nav componet 1`] = `
<nav>
  <NavLink
    activeClassName="activeLink"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="activeLink"
    to="/about"
  >
    About
  </NavLink>
  <NavLink
    activeClassName="activeLink"
    to="/contact"
  >
    Contact
  </NavLink>
</nav>
`;
